config/initializers/pusher.rb

```diff
@@ -0,0 +1,4 @@
+require 'pusher'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.